cmd/test/main.go

```diff
@@ -71,10 +71,10 @@ func (flags *TestCommandFlags) Validate() error {
 	outputValue := flags.Output
 
 	if outputValue != "" {
-		if (outputValue != "simple") && (outputValue != "json") && (outputValue != "yaml") && (outputValue != "xml") {
+		if (outputValue != "simple") && (outputValue != "json") && (outputValue != "yaml") && (outputValue != "xml") && (outputValue != "JUnit") {
```

Comment:
I am thinking about creating an array of valid outputs and iterate it instead of adding all the time `&& (outputValue != "someValidOutput")`. What do you think? this logic is repeated in serval places in this PR

Reply:
ABSOLUTLY!

I promise to do a separate refactor PR after this one, I want to keep the PR lean